### DISCUSSION TITLE
Would it be possible to use Metal / MPS on Mac M1,M2,M3

### DISCUSSION THREAD

**Discussion by mathieuchateau** [absadiki/subsai#89] [Ideas]
Hello,
As done by whispper.cpp for example, it would be great to use GPU on Mac through Metal.
I guess it depends on Model unfortunetly

**Comment by absadiki** [comment]
hello @mathieuchateau, 

Yes, it depends on the Model. 
I think `faster-whisper` supports Mac, but `whisper.cpp` offers the best support for Mac through Metal AFAIK. 
Unfortunately, I don't have a Mac to make the bindings  for `whisper.cpp`.
